PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/functionapp/config/runstate/FunctionAppDeployStateUtil.kt

```diff
@@ -156,7 +156,11 @@ object FunctionAppDeployStateUtil {
 
             functionAppStop(app, processHandler)
 
-            KuduClient.kuduZipDeploy(zipFile, app.publishingProfile, app.name(), processHandler)
+            val kuduBaseUrl = "https://" + app.defaultHostName().toLowerCase()
+                    .replace("http://", "")
```

Comment:
@maartenba, why do we need this replacement? Is that a case when `app.defaultHostName()` return full name with protocol?

Reply:
Yes, that's correct.